examples/ffi/profiles.c

```diff
@@ -3,7 +3,8 @@
 // developed at Datadog (https://www.datadoghq.com/). Copyright 2021-Present
 // Datadog, Inc.
 
-#include <ddprof/ffi.h>
+#include <libdatadog/ddcommon.h>
+#include <libdatadog/ddprof.h>
```

Comment:
Can we do this instead?

```c
#include <datadog/common.h>
#include <datadog/profiling.h>
```

Thoughts?

Reply:
Yeah agreed, removing the `lib` and `dd` prefixes might be a bit less heavy